packages/block-editor/src/components/inserter/search-items.js

```diff
@@ -39,6 +49,10 @@ function extractWords( input = '' ) {
  * @return {string} The normalized search input.
  */
 function normalizeSearchInput( input = '' ) {
```

Comment:
This feedback is only relevant to this PR in the sense that it is more noticeable/observable due to the caching, but it threw me off that the `normalizedSearch` cache includes strings that are not user search input. This is due to the `normalizeSearchInput` also processing item titles:

https://github.com/WordPress/gutenberg/blob/d90bfab807fcac371fb955d6363ea8d20f2a705e/packages/block-editor/src/components/inserter/search-items.js#L152

Could be a good opportunity to rename maybe?

```suggestion
function normalizeString( input = '' ) {
```

Reply:
I see your point and don't mind renaming as we're passing by. Done in 6ff8c2642fb9925118b85aac0d76cfa33339b03e